crates/bevy_reflect/src/lib.rs

```diff
@@ -1,4 +1,373 @@
-#![doc = include_str!("../README.md")]
+//! Reflection in Rust.
+//!
+//! [Reflection] is a powerful tool provided within many programming languages
+//! that allows for meta-programming: using information _about_ the program to
+//! _affect_ the program.
+//! In other words, reflection allows us to inspect the program itself, its
+//! syntax, and its type information at runtime.
+//!
+//! While Rust doesn't yet provide much built-in tooling for reflection,
```

Comment:
This sentence can be made much tighter.

Try "This crate adds this missing functionality to Rust."